libs/vkd3d/memory.c

```diff
@@ -1057,6 +1057,7 @@ static HRESULT vkd3d_memory_allocation_init(struct vkd3d_memory_allocation *allo
 {
     const struct vkd3d_vk_device_procs *vk_procs = &device->vk_procs;
     VkMemoryRequirements memory_requirements;
+    VkMemoryPriorityAllocateInfoEXT priority_info;
```

Comment:
:christmas_tree: ordering

Reply:
Not familiar with this - what does it mean?  Thanks.

Reply:
the length of each line is sorted from longest to shortest.